app/models/timeslot.rb

```diff
@@ -0,0 +1,4 @@
+class Timeslot < ApplicationRecord
+  belongs_to :event
+  has_many :preferences, through: :event
```

Comment:
Is the `through: :event` necessary here?

Reply:
You're right, I don't think it is.